src/openzeppelin/tests/test_account.cairo

```diff
@@ -1,38 +1,102 @@
 use array::ArrayTrait;
+use core::result::ResultTrait;
+use debug::PrintTrait;
+use option::OptionTrait;
+use starknet::class_hash::Felt252TryIntoClassHash;
 use starknet::ContractAddress;
 use starknet::contract_address_const;
+use starknet::syscalls::deploy_syscall;
 use starknet::testing::set_caller_address;
 use starknet::testing::set_contract_address;
+use starknet::testing::set_transaction_hash;
+use starknet::testing;
+use starknet::testing::set_signature;
+use traits::TryInto;
+
 use openzeppelin::account::Account;
+use openzeppelin::account::IAccountDispatcher;
+use openzeppelin::account::IAccountDispatcherTrait;
 use openzeppelin::account::ERC165_ACCOUNT_ID;
 use openzeppelin::account::ERC1271_VALIDATED;
+use openzeppelin::account::TRANSACTION_VERSION;
 use openzeppelin::account::Call;
 use openzeppelin::introspection::erc165::IERC165_ID;
 
-fn NEW_KEY() -> felt252 { 0x444444 }
-fn PUBLIC_KEY() -> felt252 { 0x333333 }
-fn OTHER() -> ContractAddress { contract_address_const::<0x222222>() }
-fn ACCOUNT_ADDRESS() -> ContractAddress { contract_address_const::<0x111111>() }
+#[derive(Drop)]
+struct ValidSignature {
```

Comment:
this seems to be more than just a signature (it includes pub/priv keys, message, etc)

Reply:
Agree the name is confusing, but the signature is valid just for that message with those keys, we don't have a signer available. Maybe **SignedTransactionHash** is a better name?